tools/sitemap.rb

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env ruby
+
+# This script creates sitemaps for Google indexing
+# As of this writing, this does not geenrate the index
+
+# Use bundler to keep dependencies local
+require 'rubygems'
+require 'bundler/setup'
+
+require 'fileutils'
+require 'sequel'
+require 'yaml'
+require 'xml-sitemap'
+
+DEST_DIR = "app/sitemaps"
```

Comment:
This wasn't actually working for me locally. I had to set the path to be /home/andy/jschol/app...etc